api_tests/lib/ledger_runner.ts

```diff
@@ -171,15 +171,8 @@ async function startBitcoinContainer(): Promise<StartedTestContainer> {
 }
 
 async function startEthereumContainer(): Promise<StartedTestContainer> {
-    return new GenericContainer("parity/parity", "v2.5.0")
-        .withCmd([
-            "--config=dev",
-            "--jsonrpc-apis=all",
-            "--unsafe-expose",
-            "--tracing=on",
-            "--logging=debug,ethcore-miner=trace,miner=trace,rpc=trace,tokio_core=warn,tokio_reactor=warn",
-            "--jsonrpc-cors='all'",
-        ])
+    return new GenericContainer("coblox/parity-poa", "v2.5.8-stable")
+        .withCmd(["--config=/parity/config/config.toml"])
```

Comment:
Why do we need to pass this?
We do we no longer pass all the other configuration options?

Reply:
All the config is now in the `config.toml` which is included in the container and we need to tell parity which config file to load.

Reply:
Can we not default to that _inside_ the image?

Reply:
You mean you would put it hardcoded into the container upon container creation and then just point to it inside the container?

Reply:
Could probably, but then it's literally a single-use container. I'd assume, once specified, you can't overwrite it from the outside.

Reply:
See this ticket here: https://github.com/coblox/docker-parity-poa/issues/3

Reply:
I am not sure I agree with the strategy.

**Inside** the docker you have a config file but you need to tell the docker from the **outside** that such file should be used.

To me a proper solution is either:
1. The docker load the file automatically
2. The docker does not contain the file but you mount a volume to pass a file you generate from a client side.

Reply:
The ticket I created and linked above is about implementing solution 1 as a follow-up of this. We can also do that first and block this PR on that if we want to have it in one go. I don't mind either as long as the current situation is not the final result :)

Reply:
Note that, as long as the file is in the default location, you can still override it from the outside with --config and a mounted volume!

Reply:
ok, @bonomat feel free to discard my "request changes".

Reply:
Let's do it properly and wait for https://github.com/coblox/docker-parity-poa/pull/5
Once the container is released I will update this PR so that it uses the default already.